apidoc/Titanium/UI/Tab.yml

```diff
@@ -224,7 +224,8 @@ properties:
     summary: The icon inset or outset for each edge.
     description: | 
         Use this property for example to center an icon without providing a title. To do that, set the
-        insets to `{top:6, bottom:-6, right:0, left:0}`.
+        insets to `{top:6, left: 0}`. The `right` and `bottom` are calculated internally to prevent the
```

Comment:
Well it's an example. If you don't set left, it will be 0 anyway. We have more critical issues for now I guess :-)